src/stylus/fablevision/trial-count.styl

```diff
@@ -0,0 +1,25 @@
+.geniblocks.trial-count
+  width: 158px
+  height: 107px
+  background-image: url('../resources/fablevision/huds/Trial_screen.png')
+  background-size: 100% 100%
+
+  .text-area
```

Comment:
I'm still trying to figure out how to name my CSS classes. Would it be better to name this something more specific like `trial-text` to give the impression that it's not a more generic text area? Or is this preferred so we could make more global changes down the line?